wordpress-mobile/WordPress-iOS
# Issue/6635 add checks for failed media

Fixes #6635 

This PR adds the checks for failed media on MediaProgressCoordinator. It also implements a method to remove failed media.

To test:
 - Start the app and create Post using Aztec
 - Put the device in offline mode
 - Add Images to the post
 - Check if they show the error state
 - Tap o X and choose Save Draft
 - See if alert warning for failed media shows up
 - Tap on the option to remove.
 - Check if it works.

Needs review: @astralbodies
